src/fragments/start/getting-started/react/auth.mdx

```diff
@@ -32,16 +32,17 @@ Now that we have our authentication service deployed to AWS, it's time to add au
 
 Open __src/App.js__ and make the following changes:
 
-1. Import the `withAuthenticator` component:
+1. Import the `withAuthenticator` component and styles:
```

Comment:
I'd prefer to update this to encourage use of the `Authenticator` component instead. It should match up with this page:
https://ui.docs.amplify.aws/components/authenticator